app/server/models/credit_transfer.py

```diff
@@ -163,8 +174,8 @@ def resolve_as_completed(self, existing_blockchain_txn=None):
         self.resolved_date = datetime.datetime.utcnow()
         self.transfer_status = TransferStatusEnum.COMPLETE
 
-        self.sender_transfer_account.balance -= self.transfer_amount
-        self.recipient_transfer_account.balance += self.transfer_amount
+        self.sender_transfer_account.add_to_balance(-1 * self.transfer_amount)
```

Comment:
seems like there should be tests for this kind of stuff :|

Reply:
Yep will add